.buildkite/release_pipeline.yaml

```diff
@@ -48,6 +56,15 @@ steps:
     command: .buildkite/scripts/build_component.sh plan-build
     agents:
       queue: habitat-release
+  
+  - label: ":linux: :habicat: :hammer_and_wrench: core/hab-plan-build"
```

Comment:
As you did with the `core/hab` build above, it would be good to either visually or textually differentiate the kernel2 builds from the "normal" builds to reduce confusion and ambiguity when looking at the overall pipeline.

Reply:
👍   I couldn't find iconography I was happy with at the time so I punted on it for a bit.  💯 agree that it's needed.

Reply:
Could maybe just use :two:?